node/tasks.py

```diff
@@ -261,9 +261,9 @@ def is_sync_point_signature_verified(
         public_key=agg_pub_key,
     )
     if not res:
```

Comment:
This section is related to the (wrong) old solution to the dynamic change of network status and should be removed